test/stripe/issuing/authorization_test.rb

```diff
@@ -32,18 +32,40 @@ class AuthorizationTest < Test::Unit::TestCase
         assert authorization.is_a?(Stripe::Issuing::Authorization)
       end
 
-      should "be approveable" do
-        authorization = Stripe::Issuing::Authorization.retrieve("iauth_123")
-        authorization.approve
-        assert_requested :post, "#{Stripe.api_base}/v1/issuing/authorizations/iauth_123/approve"
-        assert authorization.is_a?(Stripe::Issuing::Authorization)
+      context "#approve" do
```

Comment:
The convention is that `#method` is an instance method and `.method` is a class method, but here you're testing the class method in the `#approve` describe block and the instance method in the `.approve` describe block.